### DISCUSSION TITLE
pretrained_model

### DISCUSSION THREAD

**Discussion by lesdoimy** [TheLastBen/fast-stable-diffusion#522] [Q&A]
can u explain why in
def txtenc_train   --pretrained_model_name_or_path="$MODEL_NAME" \ (no $OUTPUT_DIR)
even if u resume calculates???

**Comment by TheLastBen** [comment]
Coincidence, I just fixed it, the problem is related to this : https://github.com/TheLastBen/fast-stable-diffusion/issues/501
